docs/next/guides/upgrade-and-migration/migrating-from-11.1-to-12.0.md

```diff
@@ -12,3 +12,30 @@ pageClass: migration-guide
 
 To upgrade your Handsontable version from 11.x.x to 12.x.x, follow this guide.
 
+## Step 1: Verifying your `updateSettings` calls that no longer reset the state
+
+Starting from the version 12.0.0, calling the method [`updateSettings`](@/api/core.md#updateSettings) with the `data` property no longer resets the cell meta cache and index mapper configuration. If your application relied on resetting that information, you will need to make the following change in your app.
+
+As the framework wrappers rely heavily on the [`updateSettings`](@/api/core.md#updateSettings) method to update their settings, this changes the way they react on property and state changes.
+
+### Migrating to 12.0.0
+If you need to reset the cell meta and/or index mapper information with the data update, instead of passing the `data` option to the [`updateSettings`](@/api/core.md#updateSettings) method, run [`loadData`](@/api/core.md#loadData) and provide the new dataset as an argument:
+```js
+// In 12.0.0 this will NOT reset the cell meta and index mapper configuration
+hotInstance.updateSettings({
+  data: newDataset
+});
+
+// In 12.0.0 this WILL reset the cell meta and index mapper configuration
+hotInstance.loadData(newDataset);
```

Comment:
Since this change especially affects framework wrappers, can you give a hint/link about how to use load data with the wrappers?